lib/scripts/behaviour.js

```diff
@@ -204,6 +205,35 @@ var dw_behaviour = {
                 }
             });
         }
+    },
+
+    /**
+     * Check that access to the data directory is properly secured
+     *
+     * A successful check (a 403 error was returned when loading the image) is saved
+     * to session storage and not repeated again until the next browser session. This
+     * avoids overeager security bans (see #3363)
+     */
+    securityCheck: function () {
+        var $checkA = jQuery('#security__check');
+        if (!$checkA.length) return;
+        if (sessionStorage.getItem('security-check')) {
```

Comment:
Probably make it `dw-security-check`? The namespace of sessionStorage is shared within the same domain.

Reply:
Good point. We also need to differentiate between different wikis on the same domain.